mavlink_vehicles/mavlink_vehicles.cc

```diff
@@ -54,7 +54,7 @@ const uint16_t remote_max_response_time_ms = 10000;
 const double waypoint_acceptance_radius_m = 0.01;
 const double detour_arrival_max_dist_m = 0.5;
 const double rotation_arrival_max_dif_deg = 10;
-const double is_stopped_max_speed_mps = 0.1;
+const double is_stopped_max_speed_mps = 0.3;
```

Comment:
Since this value has been changed in a more recent commit, I've just removed it from this one.